Sources/AST/ASTPassContext.swift

```diff
@@ -112,6 +112,12 @@ extension ASTPassContext {
     return functionDeclarationContext != nil || initializerDeclarationContext != nil
   }
 
+  // Whether we are visiting a node inside the rhs of an assignment.
+  public var inAssignment: Bool {
```

Comment:
Do we not want to stay consistent can call this rhsValue?

Reply:
I'm not quite sure what you mean here - what should I be being consistent with?

Perhaps `inAssignmentRhs` would be better but it's longer and not much more understandable.